docs/source/commands/persisted-queries.mdx

```diff
@@ -54,3 +54,10 @@ rover persisted-queries publish \
 2. Updates any _other_ variants that the PQL is applied to so that routers associated with those variants can fetch their updated PQL.
 
 > As with [generating manifests](/graphos/operations/persisted-queries#31-generate-persisted-queries-manifests), it's best to execute this command in your CI/CD pipeline to publish new operations as part of your app release process. This command requires your API key to have the [**Graph Admin** or **Persisted Query Publisher** role](/graphos/org/members#graph-api-key-roles); the latter is a role specifically designed to work with this command without exposing unnecessary information about your graph to users of the API key.
+
+#### Relay support
```

Comment:
Let's call out the [minimum](https://github.com/apollographql/docs#minversion) Rover version required for this:

e.g.

```suggestion

<MinVersion version="0.0.0">

#### Relay support

</MinVersion>
```